### PR TITLE
Add the ability to change the Status Type, e.g. "Watching" and "Listening" are usable now.

### PR DESCRIPTION
**Pull request by k9lego**

### CODE REVIEW

File: src/main.js [hunk 1]
```diff
@@ -69,7 +69,18 @@ function waitForGuild(guildId) {
 
 function initStatus() {
   function applyStatus() {
-    bot.editStatus(null, {name: config.status});
+    let statusType = config.statusType;
+    if (typeof statusType === "string") {
+      statusType = parseInt(statusType);
+      if (isNaN(statusType))
```
Comment (sporkwitch) [inline src/main.js]: It should be possible to greatly simplify these conditionals.  If statusType === 0 || 2 || 3, editStatus to statusType, else editStatus to 0.  Two conditionals, as opposed to the 4 currently in use.  Not a huge deal, especially on something that only happens during startup, but just from principle.

Comment (k9lego) [reply src/main.js]: Good point, those if conditions definitely can be combined.

**Comment by k9lego** [comment]
Inactivity and conflicting changes, I will open a new pull request in the future.

**Comment by Dragory** [comment]
I think the only conflict is in the README, as that had a makeover. Thank you for the pull request, and sorry for the delay on checking this!

If you open this one again or submit a new one later, can you also look into squishing the commits into 1 for the commit history?

**Comment by k9lego** [comment]
No problem It's me who has been slow lol, I'll open one in a week or so with some more error-checking to prevent bad config inputs. I will also do it in one commit. Thanks for your response!

**Comment by NicoHd105** [comment]
Ahm did it really work for you? cause i did all of your changes and it doesn't work, it's still playing btw i am using the newest vers of the bot

**Pull request closed without merging**